tsconfig.json

```diff
@@ -8,11 +8,6 @@
   "exclude": [
     "node_modules",
     "cypress",
```

Comment:
Any clue why this was excluded? this was causing the absolute import issue

Reply:
Because `cypress` tests are not written in TS?

Reply:
i left this part, i was refered to the deleted items